### DISCUSSION TITLE
9.11.01 Upgrade package fails to complete

### DISCUSSION THREAD

**Discussion by jmterhune** [dnnsoftware/Dnn.Platform#5560] [General]
Anybody else having an issue with the upgrade package for 9.11.01? I have multiple DNN installations. The upgrade hung in the dev environment for two different sites. None of the modules, libraries, or providers install and the DB scripts didn't run. If I remember correctly it was hanging when attempting to install CKEditor provider. I attempted multiple times with the same result.

**Comment by sleupold** [comment]
An incompatibility with SQL Server before 2016 has been reported and identified.
I assume, it will be fixed in DNN 9.11.2.
